braindecode/models/eeginception.py

```diff
@@ -244,17 +249,33 @@ def __init__(
                 self.n_times // prod(self.pooling_sizes))
         n_channels_last_layer = self.n_filters * len(self.scales_samples) // 4
 
-        self.add_module("classification", nn.Sequential(
```

Comment:
I would just rename `"classification"` to `"final_layer"` and keep the flatten separated (added just before, same as you did)